homeassistant/components/homekit/util.py

```diff
@@ -23,6 +23,10 @@ def validate_entity_config(values):
             raise vol.Invalid('The configuration for "{}" must be '
                               ' an dictionary.'.format(entity))
 
+        name = config.get(CONF_NAME)
```

Comment:
Can you change `key` to `entity_id` in line `19` and `20`, the `for loop`? That way we won't have this problem in the future.

Reply:
No problem, done.